rules-tests/Php80/Rector/Class_/AnnotationToAttributeRector/Fixture/Symfony/validation_integer_parameter.php.inc

```diff
@@ -8,10 +8,10 @@ final class ValidationIntegerParameter
 {
     /**
      * @Assert\Length(
-     *     min="100",
-     *     max="255",
+     *     min=100,
+     *     max=255,
      *     maxMessage="some Message",
-     *     allowed="true"
+     *     allowed=true
```

Comment:
this is changed because in Doctrine annotation semantic, using double quote means a string, as so it must be preserved in PHP as a string too.
see that integers and boolean are supported